### PR TITLE
Only show record button on chrome/firefox

### PR DESCRIPTION
**Pull request by ajpal**
Quick mitigation because recording sounds is not supported on all browsers

Chrome:
image

Safari:
image

### CODE REVIEW

**Review by epeach — APPROVED** [review]
LGTM - will this affect any eyes tests?

File: dashboard/test/ui/features/manage_assets.feature [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @no_mobile
 Feature: Manage Assets
 
+  @no_safari_yosemite
```
Comment (epeach) [inline dashboard/test/ui/features/manage_assets.feature]: Do we also need to skip IE?

Comment (ajpal) [reply dashboard/test/ui/features/manage_assets.feature]: Yes! thank you!

File: dashboard/test/ui/features/manage_assets.feature [hunk 2]
```diff
@@ -1,6 +1,8 @@
 @no_mobile
 Feature: Manage Assets
 
+  # Skip on Safari and IE. MediaRecorder is not supported so we are removing the record button.
+  @no_safari_yosemite @no_ie
```
Comment (joshlory) [inline dashboard/test/ui/features/manage_assets.feature]: Do we want Eyes tests still, to verify there _isn't_ a "Record Audio" button visible?

Comment (ajpal) [reply dashboard/test/ui/features/manage_assets.feature]: Yep, I'm working on that in another PR

**Review by epeach — APPROVED** [review]
Looks good to me

**Pull request merged**